docs/indicator_creation.md

```diff
@@ -106,3 +106,16 @@ Finally, you need to create a svg figure (e.g. with matplotlib) and save it to `
 If you have defined these three functions, your indicator is ready to go. To show how OQT uses your indicator to be displayed on the OQT Website or in your command line interface, we made a sequence diagram. 
 
 ![UML Sequence Diagram](img/UML-Sequence-Diagram.png)
+
+
+#### A note on using `scikit-learn` models
+
+Indicators can load pre-trained `scikit-learn` models from disk. Please refer to the `scikit-learn` documentation about [Model Persistence](https://scikit-learn.org/stable/modules/model_persistence.html?highlight=export) to learn what needs to be taken into account when exporting models.
+
+OQT has following minimal requirements on an `scikit-learn` model:
+- The `scitkit-learn` version with which the model is created needs to be the same version as the one OQT uses.
+- The models has to be saved to disk using the `joblib` module.
+- The cross validation score needs to be provided.
```

Comment:
I think this is rather optional, isn't it. It would be really good to provide it, but there is no technical reason for it.

Cross validation score is also a rather broad term. E.g. we are calculating 4 different scores for the building area model (r2, explained variance, mean absolute error, mean squared error.)

Reply:
Its technical not required thats true. But I thought we could right tests for each model we load into OQT and compare the validation scores calculated within OQT with the provided validation scores (Check if they are in a similar range).

But I could remove this point as well.

Reply:
Yes, we can leave it as it is. For sure it's good to be aware if these performance indicators, but maybe this is even more important on the model repository side.